src/VisualStudio/Core/Def/ManagedEditAndContinueService.vsdconfigxml

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Configuration xmlns="http://schemas.microsoft.com/vstudio/vsdconfig/2008">
+  <DefineGuid Name="ManagedEditAndContinueServiceId" Value="A96BBE03-0408-41E3-8613-6086FD494B43"/>
+  <DefineGuid Name="ManagedEditAndContinueMessageSourceId" Value="58CDF976-1923-48F7-8288-B4189F5700B1"/>
+  <ManagedComponent
+    ComponentId="ManagedEditAndContinueServiceId"
+    ComponentLevel="9991500"
+    Synchronized="true"
+    AssemblyName="Microsoft.VisualStudio.LanguageServices">
```

Comment:
Is 'Microsoft.VisualStudio.LanguageServices' something which is ngen'ed and on the probing path? If so, you probably also want `GenerateFullAssemblyName="true"` so that we try and load it using the full assembly name.